src/applications/claims-status/tests/components/ClosedClaimMessage.unit.spec.jsx

```diff
@@ -158,6 +89,7 @@ describe('<ClosedClaimMessage>', () => {
           attributes: {
             claimDate: '2023-01-01',
             closeDate: getISOString(closeDate),
+            status: 'COMPLETE',
```

Comment:
Adding the `status` field here as it's needed by the `isClaimOpen` helper function